docs/README.md

```diff
@@ -0,0 +1,288 @@
+     in that row. This is updated by the tracker, and doesn't need to be set
+     manually.
+1. Click the **Authorize script** button. If a pop-up is shown, review the
+   permissions being requested and then authorize the app to run on your behalf.
+1. Click the **Call PSI API** button to test your configuration. This should
+   result in a number of toasts being shown at the bottom right of the sheet,
+   with the final toast saying to check the results sheet.
+
+   <img src="./imgs/psi_tracker_toast.webp" alt="Google Sheets notification toast saying 'Received Data. Parsing information - Check the Results sheet'." width="25%">
+
+1. If the Results sheet is properly populated after clicking the **Call PSI
+   API** button, click the **Set PSI daily trigger** button to start daily
+   measurement.
+
+   <img src="./imgs/psi_tracker_results.webp" alt="Screenshot of the Performance PSI Tracker Results tab with metrics for multiple days." width="50%">
+
+Following these steps should result in the Results tab being updated daily with
+new performance metrics. The rest of the settings on the sheet are for advanced
+users and detailed below.
+
+## Using the Performance PSI Tracker
+
+Once everything is set up, the Results tab is populated daily providing a steady
+stream of performance data.
+
+The CrUX-related columns in the Results are automatically color coded for you
+based on the [official thresholds from web.dev](https://web.dev/vitals/).
+<span style="background: #b7e1cd">Green</span> is good;
+<span style="background: #fff2cc">Yellow</span> is Needs Improvement;
+<span style="background: #f4cccc">Red</span> is Poor.
+
+To review the progress of a single page, you can add a data filter to the first
+row of the Results tab. Depending on how you have labeled your pages in the
+Config, you can either use the URL column or the Label column as the filter. If
+you wish to review multiple pages, the page labels are often a better
+choice. Please refer to the [Google Sheets documentation on
+filtering](https://support.google.com/docs/answer/3540681) for more
+information. Using just the color coding, you should be able to see if your
+pages are regressing in a significant way.
+
+If you are planning on using the Lighthouse metrics for tracking, we suggest
+adding your own conditional formatting to the relevant columns to make it easier
+to see how your site is performing. Please see the [Google Sheets documentation
+on conditional
+formatting](https://support.google.com/docs/answer/78413?hl=en&co=GENIE.Platform%3DDesktop&oco=0)
+for more information.
+
+> [!WARNING] 
+> There are two buttons at the bottom of the Config worksheet that
+> can be destructive to any changes you've made to the Tracker.
+>
+> **Reset** will replace all of the URLs entered in the Config worksheet with a
+> standard set of URLs from web.dev.
+>
+> **Delete** will delete all of the data in the Results worksheet.
+
+## Visualizing Your Metrics
+
```

Comment:
As mentioned above, I feel like it would be useful to have a short intro describing that there are two ways to visualize data, and what each of them gives you.

eg: 
There are two main ways to visualize your data:

* **Google Sheets:** Perfect for those who need a quick and easy way to visualize their data. It may not be suitable for complex data visualization needs.
* **Looker Studio:** This is a more powerful and feature-rich tool that allows you to create dashboards and interactive visualizations. It's perfect for those who need a more robust solution, but it requires a lengthier setup.